entrypoint.sh

```diff
@@ -32,23 +32,21 @@ echo "###"
 echo "# Game server port is '${FACTORIO_PORT}'"
 echo "###"
 
-if [ -z $FACTORIO_RCON_PORT ]
-then
-  factorio_command="$factorio_command --rcon-port ${FACTORIO_RCON_PORT:-27015}"
-  echo "###"
-  echo "# RCON port is '${FACTORIO_RCON_PORT}'"
-  echo "###"
-fi
+factorio_command="$factorio_command --rcon-port ${FACTORIO_RCON_PORT:-27015}"
+echo "###"
+echo "# RCON port is '${FACTORIO_RCON_PORT}'"
```

Comment:
This can be undefined as-is. You'll want to do something like line 29 in breaking that variable out.